src/main/java/org/broadinstitute/hellbender/tools/funcotator/dataSources/gencode/GencodeFuncotationFactory.java

```diff
@@ -1515,13 +1528,13 @@ static SequenceComparison createSequenceComparison(final VariantContext variant,
         // Get the strand-corrected alleles from the inputs.
         // Also get the reference sequence for the variant region.
         // (spanning the entire length of both the reference and the variant, regardless of which is longer).
-        final Allele refAllele = FuncotatorUtils.getStrandCorrectedAllele(variant.getReference(), transcript.getGenomicStrand());
-        final Allele altAllele = FuncotatorUtils.getStrandCorrectedAllele(alternateAllele, transcript.getGenomicStrand());
-        final String referenceBases = getReferenceBases(variant.getReference(), alternateAllele, reference, transcript.getGenomicStrand());
+        final Allele                                        refAllele      = FuncotatorUtils.getStrandCorrectedAllele(variant.getReference(), transcript.getGenomicStrand());
+        final Allele                                        altAllele      = FuncotatorUtils.getStrandCorrectedAllele(alternateAllele, transcript.getGenomicStrand());
```

Comment:
There's no marker type for strand-corrected Alleles, so we still have the same problem with Alleles as we did with reference bases. Recommend that you introduce an empty subclass of `Allele` to serve as a marker, and use that whenever a method expects the alleles to be strand-corrected.

Reply:
This is a good idea.  I've created the base class type and issue #5354 to track this.